<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>299</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Scales</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Vertical Scale</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="checkBoxAutoDetection">
            <property name="text">
             <string>Specify graph range:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel9_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Minimum value:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxMinValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KDoubleNumInput" name="spinBoxMinValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>49999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
            <property name="maxValue" stdset="0">
             <number>49999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="textLabel9_2_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Maximum value:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxMaxValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="KDoubleNumInput" name="spinBoxMaxValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>50000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
            <property name="maxValue" stdset="0">
             <number>50000</number>
            </property>
            <property name="minValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Horizontal Scale</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="textLabel8_2">
            <property name="text">
             <string>Pixels per time period:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KIntSpinBox" name="spinBoxPixel">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
            <property name="maxValue" stdset="0">
             <number>50</number>
            </property>
            <property name="minValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Grid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox8">
         <property name="title">
          <string>Lines</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBoxVLines">
              <property name="text">
               <string>Vertical lines</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_2_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Distance:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>spinBoxDistance</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KIntSpinBox" name="spinBoxDistance">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="value">
               <number>30</number>
              </property>
              <property name="maxValue" stdset="0">
               <number>120</number>
              </property>
              <property name="minValue" stdset="0">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxVLinesScroll">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Vertical lines scroll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxHLines">
            <property name="text">
             <string>Horizontal lines</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox9">
         <property name="title">
          <string>Text</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkBoxLabels">
            <property name="text">
             <string>Show axis labels</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="textLabel3_2">
              <property name="text">
               <string>Font size:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>spinBoxFontSize</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KIntSpinBox" name="spinBoxFontSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>8</number>
              </property>
              <property name="maxValue" stdset="0">
               <number>0</number>
              </property>
              <property name="minValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Sensors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBoxIncoming">
           <property name="text">
            <string>Show incoming traffic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KColorButton" name="kColorButtonIncoming">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="checkBoxOutgoing">
           <property name="text">
            <string>Show outgoing traffic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KColorButton" name="kColorButtonOutgoing">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>185</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>checkBoxAutoDetection</tabstop>
  <tabstop>spinBoxMinValue</tabstop>
  <tabstop>spinBoxMaxValue</tabstop>
  <tabstop>spinBoxPixel</tabstop>
  <tabstop>checkBoxVLines</tabstop>
  <tabstop>spinBoxDistance</tabstop>
  <tabstop>checkBoxVLinesScroll</tabstop>
  <tabstop>checkBoxHLines</tabstop>
  <tabstop>checkBoxLabels</tabstop>
  <tabstop>spinBoxFontSize</tabstop>
  <tabstop>checkBoxIncoming</tabstop>
  <tabstop>kColorButtonIncoming</tabstop>
  <tabstop>checkBoxOutgoing</tabstop>
  <tabstop>kColorButtonOutgoing</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAutoDetection</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxMinValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoDetection</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxMaxValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxVLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxDistance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxVLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxVLinesScroll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoDetection</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel9_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoDetection</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel9_2_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxVLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_2_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxIncoming</sender>
   <signal>toggled(bool)</signal>
   <receiver>kColorButtonIncoming</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxOutgoing</sender>
   <signal>toggled(bool)</signal>
   <receiver>kColorButtonOutgoing</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
